DCA/adapter.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+
+from PIL import Image
+import argparse
+from pathlib import Path
+
+from adapter_base import AdapterBase
+from utils.tools import *
```

Comment:
[Why import star in Python is a bad idea](https://www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/)